gammapy/datasets/tests/test_fermi.py

```diff
@@ -33,7 +32,7 @@ def test_psf(self):
         energy = Quantity(100, 'GeV')
         fraction = 0.68
         angle = psf.containment_radius(energy, fraction)
-        assert_quantity(angle, Angle('0.1927459865412511 deg'))
+        assert_quantity_allclose(angle, Angle('0.1927459865412511 deg'))
```

Comment:
Please change to this format:
```python
Angle(0.1927459865412511, 'deg')
```
It's the same, but let's try to be consistent in our use of `Angle`, `Quantity`, ... if you come across other similar cases, feel free to change those to this format as well.